runtime/queries/agda/highlights.scm

```diff
@@ -62,20 +71,16 @@
   "infixl"
   "infixr"
   "record"
-]
-@keyword
-
-;;;(expr 
-;;;	f_name: (atom) @function)
-;; Brackets
+] @keyword
 
+;(expr 
+;	f_name: (atom) @function)
```

Comment:
```suggestion
; (expr f_name: (atom) @function)
```

Reply:
I don't think commented code is -- or can be -- formatted.

Reply:
Not automatically but we might as well format the ones we notice.

Reply:
Not in this PR; it's big enough as it is.